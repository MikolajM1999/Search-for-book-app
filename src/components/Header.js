import React from 'react'

const Header = () => (
   <header className="header page__header">
      <h1 className="header__title">Search-For-Book-App</h1>
   </header>
)

export default Header