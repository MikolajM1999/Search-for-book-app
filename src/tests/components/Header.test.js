import React from 'react'
import { shallow } from 'enzyme'
import Header from '../../components/Header'

test('Header renders without crashing', () => {
   shallow(<Header />)
})

test('should render Header correctly', () => {
   const wrapper = shallow(<Header />)
   expect(wrapper).toMatchSnapshot()
})