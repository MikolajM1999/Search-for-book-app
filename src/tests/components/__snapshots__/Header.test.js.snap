// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header
  className="header page__header"
>
  <h1
    className="header__title"
  >
    Search-For-Book-App
  </h1>
</header>
`;
